packages/aws-cdk/lib/api/hotswap/step-functions.ts

```diff
@@ -0,0 +1,113 @@
+import * as cfn_diff from '@aws-cdk/cloudformation-diff';
+import { ISDK } from '../aws-auth';
+import { ChangeHotswapImpact, ChangeHotswapResult, HotswapOperation, ListStackResources, stringifyPotentialCfnExpression } from './common';
+
+export function isHotswappableStepFunctionChange(
```

Comment:
I think we should replace all mentions of "step functions" with "state machines" everywhere in this file in the names, as that's the actual resource being hotswapped.

I understand they're technically "StepFunctions State Machines", but I feel like that's too long for the nams.